os/board/rtl8730e/src/component/bluetooth/api/ipc/bt_ipc_parameter_conversion.c

```diff
@@ -2260,6 +2260,7 @@ uint32_t rtk_br_get_act_ret_param_size(uint16_t act)
 //	}
 //
 //	return max_size;
+	return RTK_BT_OK;
```

Comment:
Is this function necessary with always OK even it requires size get?

Reply:
I have modified the return values to max size.